web/rootfs/etc/cont-init.d/10-config

```diff
@@ -10,8 +10,12 @@ mkdir -p \
 # generate keys (maybe)
 if [[ $DISABLE_HTTPS -ne 1 ]]; then
     if [[ $ENABLE_LETSENCRYPT -eq 1 ]]; then
-        mkdir -p /config/acme.sh
-        sh /opt/acme.sh --install \
+        mkdir -p /config
+        cd /config/
+        git clone git@github.com:acmesh-official/acme.sh.git
```

Comment:
Rather than pulling it from git, please use https://github.com/acmesh-official/acme.sh/archive/refs/tags/3.0.6.tar.gz and `wget` which is already in the base image.